uniprotkb-rest/src/main/resources/publication.facet.properties

```diff
@@ -11,6 +11,7 @@ facet.publication.types.value.3=UniProtKB unreviewed (TrEMBL)
 
 facet.publication.categories.label=Category
 facet.publication.categories.allowmultipleselection=true
+facet.publication.categories.limit=15
 
```

Comment:
Leo, correct me if i am wrong. I can see there are 10 enums in PublicationCategory. If we include Unclassified then the count will become 11. Should we not set `facet.publication.categories.limit=11`.
Could you please add a test case to verify if we are getting more than 5 facets?